scripts/zones/Carpenters_Landing/mobs/Para.lua

```diff
@@ -0,0 +1,80 @@
+-----------------------------------
+-- Area: Carpenters' Landing
+-- NM: Para
+-- Quest: Elderly Pursuits
+-----------------------------------
+local ID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+local checkIfShouldClone = function(hpp, skillId)
+    if
+        (xi.mobSkill.QUEASYSHROOM == skillId or
+        xi.mobSkill.NUMBSHROOM == skillId or
+        xi.mobSkill.SHAKESHROOM == skillId) and
+        hpp < 50
+    then
+        return true
+    else
+        return false
+    end
+end
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 180) -- 3 minutes
```

Comment:
Based on the capture by Siknoz Para is also immune to Light Sleep - https://youtu.be/beJ9ca8aZYM?si=1xqFcWDeMtQq2biG&t=1099

And 40% MDT - https://youtu.be/beJ9ca8aZYM?si=iByN6ePE3YtNjbTl&t=1175